[Error] Bootstrapping gives `expected type '[]const u8', found '.@build.struct:8:14'` on Zig 0.8.1

Before I start: Yes, I checked this out recursively. I suspect if I didn't I'd still hit this error before the obvious problem happens.

Here's the compile error I get:

```
$ zig build -Drelease-safe
./build.zig:71:20: error: expected type '[]const u8', found '.@build.struct:8:14'
    lib.addPackage(clap);
                   ^
./build.zig:8:14: note: .@build.struct:8:14 declared here
    .path = .{ .path = "libs/zig-clap/clap.zig" },
             ^
```

Commenting out that line gives a different error:

```
$ zig build -Drelease-safe
./build.zig:55:24: error: no member named 'path' in '[]const u8'
            .path = .{ .path = "libs/mecha/mecha.zig" },
                       ^
./build.zig:72:20: note: referenced here
    lib.addPackage(version);
                   ^
```

... I think I've somehow rubber-ducked what the fix will be, if I'm right in that then I'll have a PR for this.

EDIT: Yep, got the fix:

```
sed -i -e 's/\.{ \.path = \(".*"\) }/\1/g' build.zig
```

And now there's issues with specific dependencies not supporting Zig 0.8.1. But this should fix gyro itself.

--- a/README.md
+++ b/README.md
@@ -74,8 +74,10 @@ a branch, tag, or commit hash, if not specified, gyro uses the default branch.
 ## Installation
 
 In order to install gyro, all you need to do is extract one of the [release
-tarballs](https://github.com/mattnite/gyro/releases) for your system and add
-the single static binary to your PATH.
+tarballs](https://github.com/mattnite/gyro/releases) for your system and add the
+single static binary to your PATH. Gyro follows the master branch of the zig
+compiler, if you are using zig version 0.8.1 or earlier you need to use gyro
+0.2.3.
 
 ### Building
 

